master/buildbot/process/build.py

```diff
@@ -533,22 +538,21 @@ def buildFinished(self, text, results):
         state.
 
         It takes two arguments which describe the overall build status:
-        text, results. 'results' is one of SUCCESS, WARNINGS, or FAILURE.
+        text, results. 'results' is one of the possible results (see buildbot.status.results).
```

Comment:
Somewhere in the code you use @type and @param to describe params, why not doing it here also?
With sphynx, you should be also able to do :ref:`buildbot.status.results`